### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by maxkadushkin**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,11 +10,16 @@
 * Fixed some security issues
 * Fixed opening files with size more than 100MB
```
Comment (LuckySpecialist) [inline CHANGELOG.md:11]: * Fixed opening files with size more than 100 MB

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,11 +10,16 @@
...
 * Fixed some issues with composite input
-* Fixed issues preventing work application on MacOS version < 12.0
```
Comment (LuckySpecialist) [inline CHANGELOG.md:13]: * Fixed working with blocked text fields in dialog boxes

File: CHANGELOG.md [hunk 3]
```diff
@@ -23,18 +28,21 @@
 * Fixed equation transformation for Linear/Professional mode switching from
   context menu
 * Fixed print issue for Header/Footer
+* Fixed crashing of the editor after creating file from password protected XLTX
 
 #### Presentation Editor
 
 * Fixed error on click Head & Footer in Insert tools panel
```
Comment (LuckySpecialist) [inline CHANGELOG.md:35]: * Fixed error on click Header & Footer in Insert tools panel

File: CHANGELOG.md [hunk 4]
```diff
@@ -23,18 +28,21 @@
...
 * Fixed crashing in the Reporter mode on changing slide from end demonstration
+* Fixed saving presentations with media files for compatibility with other editors
 
 #### PDF Editor
 
 * Fixed saving document through "Save copy"
 * Fixed incorrect processing editor's config (editorType)
 * Fixed issue for plugin "Send" when there is no email client installed
 * Fixed issue related to input disabled on creating
```
Comment (LuckySpecialist) [inline CHANGELOG.md:44]: delete line

**Review by LuckySpecialist — COMMENTED** [review]
* Fixed print options (range) tuning for documents and presentations
* Fixed some security issues
* Fixed opening files with size more than 100 MB
* Fixed some issues with composite input

**Review by agolybev — APPROVED** [review]

**Pull request merged**